docs/02-for-app-authors/03-metainfo-guidelines/01-quality-guidelines.md

```diff
@@ -30,6 +30,8 @@ Use Flathub verification to indicate that you are the author of the app and that
 
 Apps should set a [brand color](/docs/for-app-authors/metainfo-guidelines/#brand-color) (in two variants, light and dark). This color is used by Flathub and native app store clients on banners, app pages, etc.
```

Comment:
```suggestion
Apps should set a `primary` [brand color](/docs/for-app-authors/metainfo-guidelines/#brand-color) (in two variants, light and dark). This color is used by Flathub and native app store clients on banners, app pages, etc.
```

Reply:
Not yet, but the spec has you set the `type` to `primary`

Reply:
Ah, hmm. Without furher context that might be a bit confusing, no? Primary kind of implies there are other types.

Reply:
It is mentioned in the appstream docs, but we should probably just mention it too https://www.freedesktop.org/software/appstream/docs/chap-Metadata.html#tag-branding

Reply:
Not sure we need to mention it in the text since it's in the code example anyway, but no super strong opinion.